### PR TITLE
fix: update phoenix_analytics path in Mix.install

### PR DESCRIPTION
**Pull request by cbuctok**
Changed the path of phoenix_analytics from a relative external path to the local directory to ensure proper dependency loading.

### CODE REVIEW

**Comment by lalabuy948** [comment]
Thank you for your contribution @cbuctok ! Since you are on that topic, could you please add `ip: {0, 0, 0, 0}` to `endpoint_options` as at the moment vegeta cannot reach it. 

There is a plan to build nice dev environment if this library will pick up. 
Using something like: https://github.com/wojtekmach/mix_install_examples/blob/main/phoenix.exs

**Comment by cbuctok** [comment]
Thanks for the feedback! I'll go ahead and add `ip: {0, 0, 0, 0}` to the `endpoint_options` to ensure compatibility with `vegeta`. That should help with reaching the endpoint - I use it for my local env but was not sure if it is the way to go.

File: priv/repo/dev.exs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env elixir
 Mix.install([
   {:phoenix_playground, "~> 0.1.5"},
-  {:phoenix_analytics, path: "../phoenix_analytics", force: true}
+  {:phoenix_analytics, path: ".", force: true}
```
Comment (lalabuy948) [inline priv/repo/dev.exs:4]: > [!TIP]
> Nice!

**Comment by lalabuy948** [comment]
@cbuctok you can target 01.2 for this one as well!

**Pull request merged**